docs/content/en/docs/installation/runtime-hooks.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Configure Runtime Hooks"
+linkTitle: "Runtime Hooks"
+weight: 3
+description: "Configure Runtime Hooks"
+---
+
+See [Tetragon Rutime Hooks]({{< ref "/docs/concepts/runtime-hooks" >}}), for an introduction to
+the topic.
+
+We use `minikube` as the example platform because it supports both `cri-o` and `containerd`. Also,
+at the time of this writing, no images that support this have been released, so we build images
+locally:
+
+```shell-session
+$ make image image-operator image-rthooks # build images within a tetragon checked out git repo
+```
+
+## CRI-O
+
+### Setup Minikube
+
+```shell-session
+$ minikube start --driver=kvm2 --container-runtime=cri-o
+$ minikube image load cilium/tetragon:latest
+$ minikube image load cilium/tetragon-operator:latest
+$ minikube image load cilium/tetragon-rthooks:latest
+$ minikube image ls | grep tetragon
+localhost/cilium/tetragon:latest
+localhost/cilium/tetragon-rthooks:latest
+localhost/cilium/tetragon-operator:latest
+```
```

Comment:
Following k8s.io convention and making the command copy-pastable:
````suggestion
```shell
minikube start --driver=kvm2 --container-runtime=cri-o
minikube image load cilium/tetragon:latest
minikube image load cilium/tetragon-operator:latest
minikube image load cilium/tetragon-rthooks:latest
minikube image ls | grep tetragon
```

The output should be similar to:

```
localhost/cilium/tetragon:latest
localhost/cilium/tetragon-rthooks:latest
localhost/cilium/tetragon-operator:latest
```
````